test_scripts/AppServices/PerformAppServiceInteraction/012_M2H_GENERIC_ERROR.lua

```diff
@@ -0,0 +1,76 @@
+---------------------------------------------------------------------------------------------------
+--  Precondition: 
+--  1) Application with <appID> is registered on SDL.
+--  2) Specific permissions are assigned for <appID> with PerformAppServiceInteraction
+--  3) HMI has published a MEDIA service
+--
+--  Steps:
+--  1) Application sends a PerformAppServiceInteraction RPC request with HMI's serviceID
+--
+--  Expected:
+--  1) SDL forwards the PerformAppServiceInteraction request to the HMI as AppService.PerformAppServiceInteraction
+--  2) HMI sends a AppService.PerformAppServiceInteraction response (SUCCESS) to Core with a serviceSpecificResult
+--  3) SDL forwards the response to Application as PerformAppServiceInteraction
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/AppServices/commonAppServices')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local manifest = {
+  serviceName = "HMI_MEDIA_SERVICE",
+  serviceType = "MEDIA",
+  allowAppConsumers = true,
+  rpcSpecVersion = config.application1.registerAppInterfaceParams.syncMsgVersion,
+  mediaServiceManifest = {}
+}
+
+local rpc = {
+  name = "PerformAppServiceInteraction",
+  hmiName = "AppService.PerformAppServiceInteraction",
+  params = {
+    originApp = config.application1.registerAppInterfaceParams.fullAppID,
+    serviceUri = "hmi:sample.service.uri"
+  }
+}
+
+local expectedResponse = {
+  success = false,
+  resultCode = "GENERIC_ERROR"
+}
+
+local function PTUfunc(tbl)
+  tbl.policy_table.app_policies[common.getConfigAppParams(1).fullAppID] = common.getAppServiceConsumerConfig(1);
+end
+
+--[[ Local Functions ]]
+local function processRPCSuccess(self)
+  local mobileSession = common.getMobileSession()
+  local service_id = common.getAppServiceID(0)
+  local requestParams = rpc.params
+  requestParams.serviceID = service_id
+  local cid = mobileSession:SendRPC(rpc.name, requestParams)
+  EXPECT_HMICALL(rpc.hmiName, requestParams)
```

Comment:
Maybe add a `--Do not respond` before the EXPECT_HMICALL

Reply:
Fixed in https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2180/commits/0412e9deb1787fac29a0742919856427f1eabf19